frontend/components/color_encoding_to_sound/PaletteColor.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import SliderInstrument from "../instruments/SliderInstrument";
+
+class PaletteColor extends React.Component {
+    render() {
+        const selected = this.props.selected;
+        const color = this.props.color;
+        const styleString = `background-color: rgb(${color.r},${color.g},${color.b})`;
```

Comment:
Nitpick: this variable is unused -- should've been caught by the linter. Guessing it became unnecessary once the `style` var took on more stuff and needed to become an object.